docs/resources/cce_node_pool_v3.md

```diff
@@ -113,6 +113,11 @@ The `root_volume` block supports:
 
 * `volumetype` - (Required, String) Specifies the disk type.
 
+* `kms_key_id` - (Optional, String) Specifies the KMS key ID. This is used to encrypt the volume.
```

Comment:
it seems that the docs of `cce_node_v3` should also be updated

Reply:
i've updated the cce_node_v3 docs as well